### PR TITLE
MMT-1445: make variables use UmmPreview* classes

### PR DESCRIPTION
**Pull request by charleshuang80**
* make variables and variable drafts use UmmPreview* classes
* add UmmPreviewMultiItems
* fix preview rendering
* add links in appropriate places
* address preview comments for Umm Var
* modify tests

### CODE REVIEW

File: app/assets/stylesheets/components/_metadata-fields.scss [hunk 1]
```diff
@@ -51,6 +51,13 @@
 }
 
 .preview {
+  /*
+  not sure how, but this gets translated into:
```
Comment (macrouch) [inline app/assets/stylesheets/components/_metadata-fields.scss]: https://github.com/thoughtbot/neat/blob/v1.8.0/app/assets/stylesheets/grid/_span-columns.scss

File: lib/json_schema_form/umm_preview_elements/umm_preview_form.rb [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# :nodoc:
+class UmmPreviewForm < UmmPreview
```
Comment (macrouch) [inline lib/json_schema_form/umm_preview_elements/umm_preview_form.rb]: Might want to add more comments to the methods in this file

File: lib/json_schema_form/umm_preview_elements/umm_preview_science_keyword.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    Category
+    Topic
+    Term
+    VariableLevel1
+    VariableLevel2
+    VariableLevel3
+    DetailedVariable
+  ).freeze
+
+  # def initialize(data)
```
Comment (macrouch) [inline lib/json_schema_form/umm_preview_elements/umm_preview_science_keyword.rb]: should this be commented out? if so, remove it. Also, `umm_preview_service_keyword.rb` has it in, should both of them be removed? I think if the only thing they are doing is calling `super`, the answer is remove both

File: lib/json_schema_form/umm_preview_elements/umm_preview_sidebar.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# :nodoc:
```
Comment (macrouch) [inline lib/json_schema_form/umm_preview_elements/umm_preview_sidebar.rb]: I don't think we ended up needing this file at all

File: lib/assets/schemas/variables/umm-var-preview.json [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+        }, {
+          "key": "Units"
+        }, {
+          "key": "DataType"
+        }, {
+          "key": "Scale"
+        }, {
+          "key": "Offset"
+        }, {
+          "key": "ValidRanges/index_id", // don't need title because titleize is fine
```
Comment (charleshuang80) [inline lib/assets/schemas/variables/umm-var-preview.json]: remove comment

**Pull request merged**